atqa-framework/utils/checkFile.ts

```diff
@@ -0,0 +1,16 @@
+import fs from 'fs-extra';
+import { File } from '../interfaces/File.interface';
+import { ClockUnit } from './clockUnit';
+
+const clockUnit = new ClockUnit();
+
+export const checkFile = async (file: File): Promise<void> => {
```

Comment:
This doesn't seem to be used. And looking at the code it does nothing useful actually.

Reply:
This one for the next stage, it will be used in future tests. Let's keep it.

Reply:
You can keep it in a separate branch then